PARQUET-1260: Add Zoltan Ivanfi's code signing key to the KEYS file

zivanfi opened the pull request.

xhochy commented:
We seem to be out of sync with the keys in SVN and in the many git repos we have in parquet. 

There is a KEYS per parquet-{mr/format/cpp} and https://dist.apache.org/repos/dist/dev/parquet/KEYS and https://dist.apache.org/repos/dist/release/parquet/KEYS

I wanted to commit this change here to SVN but then we would have other changes. Can someone take care of merging all the KEYS together? I can then do the sync to SVN.

zivanfi commented:
When I added my key to the KEY files, they were already diverged at that point and I couldn't figure out which one should "win", so I just added my key to both of them, but I didn't cause any further inconsistencies (I believe).
